### PR TITLE
Issue 380 - /whitepaper route added as static HTML

### PR DESCRIPTION
**Pull request by Phasor**
Had a few CSS conflict issues with the previous PR for this issue. Fixed on this branch.

### CODE REVIEW

File: src/components/Toolbar/Toolbar.tsx [hunk 1]
```diff
@@ -70,6 +70,11 @@ const Toolbar: FC<ToolbarProps> = ({ onButtonClick }) => {
           text={t("common.join")}
           onClick={() => onToolbarButtonClick("join")}
         />
+        <ToolbarButton
```
Comment (piersss) [inline src/components/Toolbar/Toolbar.tsx:73]: Use react-router for internal links in the app. So modify ToolbarButton so it accepts link as property and use react-router accordingly.

File: src/components/Whitepaper/Whitepaper.tsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React, { FC } from "react";
+
+import htmlContent from "./WhitepaperContent";
+
+const Whitepaper: FC = () => {
+  return (
+    <>
+      <div dangerouslySetInnerHTML={{ __html: htmlContent }} />
+    </>
```
Comment (piersss) [inline src/components/Whitepaper/Whitepaper.tsx:9]:
```
return (
  <div dangerouslySetInnerHTML={{ __html: htmlContent }} />
);
```

File: src/components/Toolbar/Toolbar.tsx [hunk 3]
```diff
@@ -70,6 +70,11 @@ const Toolbar: FC<ToolbarProps> = ({ onButtonClick }) => {
...
+          iconName="learn"
+          text="Whitepaper"
```
Comment (piersss) [inline src/components/Toolbar/Toolbar.tsx:75]: Add a translation for whitepaper with `useTranslation`. Ask Don or Brigand to help you add it to poeditor.

Also the word whitepaper doesn't fit. Please contact design how to solve this.
Screenshot 2022-01-31 at 16 53 54

**Review by piersss — CHANGES_REQUESTED** [review]
Hey again phasor!

I guess we could use pure css instead of styled components for this if the other developers are ok with it.

One nitpicky thing: could you use camelCase for the files in /public?

Also the toolbar was not designed for 7 buttons, so it doesn't fit anymore. But that is something beyond the scope of this ticket so I could fix that later.

File: src/components/Toolbar/Toolbar.tsx (hunk 1 above)
Comment (piersss) [reply src/components/Toolbar/Toolbar.tsx:73]: No need for toolbar button anymore. So this is resolved.

File: src/components/Toolbar/Toolbar.tsx (hunk 3 above)
Comment (piersss) [reply src/components/Toolbar/Toolbar.tsx:75]: No need for toolbar button anymore. So this is resolved.

File: src/components/Whitepaper/Whitepaper.tsx (hunk 2 above)
Comment (Phasor) [reply src/components/Whitepaper/Whitepaper.tsx]: Updated thanks

**Review by piersss — APPROVED** [review]
All good!

**Pull request merged**